src/services/ErrorHandler.php

```
@@ -52,69 +38,59 @@ public function handleError($exception): craft\web\Response|string|array
+        return $this->renderError(Craft::$app->getRequest(), $statusCode, $exception);
     }
 
-    /**
-     * Renders an error template for any status code.
-     *
-     * @param \yii\web\Request $request
-     * @param int $statusCode
-     * @return craft\web\Response|string|array
-     */
-    public function renderError($request, int $statusCode, $exception = null): craft\web\Response|string|array
+    public function renderError($request, int $statusCode, $exception = null): \craft\web\Response
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)